seed/views/main.py

```diff
@@ -2376,16 +2377,18 @@ def save_gb_request_info(request):
     if time_type == 'date':
         last_date = yesterday.strftime(date_pattern)
         if loopback_flag == 'Y':
-            very_first_date = datetime.strptime('1/1/2014', '%m/%d/%Y').date()
+            very_first_date = datetime.strptime('3/1/2014', '%m/%d/%Y').date()
```

Comment:
I will remove the hard code start date later, currently the GreenButton
service I'm using sometimes returns '204 no content' if the start date is
set too early. I will try to load the backfill data month by month
backwardly until there is no data anymore.

On Thu, Mar 3, 2016 at 4:09 PM, Nicholas Long <notifications@github.com>
wrote:

> In seed/views/main.py
> <https://github.com/SEED-platform/seed/pull/817#discussion_r54948663>:
>
> > @@ -2376,16 +2377,18 @@ def save_gb_request_info(request):
> >      if time_type == 'date':
> >          last_date = yesterday.strftime(date_pattern)
> >          if loopback_flag == 'Y':
> > -            very_first_date = datetime.strptime('1/1/2014', '%m/%d/%Y').date()
> > +            very_first_date = datetime.strptime('3/1/2014', '%m/%d/%Y').date()
>
> what if we want to import files older than 3/1/2014?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/817/files#r54948663>.
>